.devcontainer/devcontainer.json

```diff
@@ -0,0 +1,31 @@
+// For format details, see https://aka.ms/devcontainer.json. For config options, see the
+// README at: https://github.com/devcontainers/templates/tree/main/src/anaconda
+{
+	"name": "Anaconda (Python 3)",
+	"build": {
+		"context": "..",
+		"dockerfile": "Dockerfile"
+	},
+	"features": {
+		"ghcr.io/devcontainers/features/docker-in-docker:2": {},
+		"ghcr.io/devcontainers/features/python": {
+			"installJupyterlab": true
+		},
+		"ghcr.io/devcontainers/features/nvidia-cuda:1": {
+			"version": 11.3
+		},
+		"ghcr.io/devcontainers-contrib/features/flake8:2": {}
+	},
+	// Features to add to the dev container. More info: https://containers.dev/features.
+	// "features": {},
+	// Use 'forwardPorts' to make a list of ports inside the container available locally.
+	// "forwardPorts": [],
+	// Install some dependencies on GitHub actions
+	"onCreateCommand": "./.devcontainer/onCreateCommand.sh",
+	// Use 'postCreateCommand' to run commands after the container is created.
+	"postCreateCommand": "./.devcontainer/postCreateCommand.sh"
+	// Configure tool-specific properties.
+	// "customizations": {},
+	// Uncomment to connect as root instead. More info: https://aka.ms/dev-containers-non-root.
```

Comment:
Creating a codespace on 32GB failed for me due to insufficient storage, can we add this config?

```
"hostRequirements": {
		"storage": "64gb"
	}
```

Reply:
thanks I will add it